src/components/Calculator.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
- [ ] @ambrose-kibet it's not good practice to disable linter checks in your file, kindly fix the linter errors instead of disabling the checks.